src/components/file/file.tsx

```diff
@@ -80,6 +81,14 @@ export class File {
     @Prop({ reflect: true })
     public disabled: boolean = false;
 
+    /**
+     * The accepted file types
+     *
+     * @see https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/file#unique_file_type_specifiers
```

Comment:
Really nice and necessary to have this resource in the docs string. I noticed that I couldn't click it in the documentation. could that work if you replace `@see` with `@link`?